### PR TITLE
Add household median income to pvtree menu

### PR DESCRIPTION
**Pull request by shifucun**
image

### CODE REVIEW

File: static/data/hierarchy_top.json [hunk 1]
```diff
@@ -355,15 +355,24 @@
...
   "e": "Household",
-  "c": 680,
+  "c": 690,
   "cd": [
    {
     "sv": [
      "Count_Household"
     ],
-    "l": "Household",
+    "l": "Count",
```
Comment (beets) [inline static/data/hierarchy_top.json:364]: perhaps we can call this "population" instead?

File: static/data/hierarchy_top.json [hunk 2]
```diff
@@ -355,15 +355,24 @@
...
     "t": "v",
-    "e": "Household",
+    "e": "Count",
+    "c": 1
+   },
+   {
+    "sv": [
+     "Median_Income_Household"
+    ],
+    "l": "Median Income",
```
Comment (beets) [inline static/data/hierarchy_top.json:373]: Median household income, so it won't be confused with median individual income. is this the string that will be used for the chips as well?

**Review by beets — APPROVED** [review]

File: static/data/hierarchy_top.json (hunk 1 above)
Comment (shifucun) [reply static/data/hierarchy_top.json]: Done

File: static/data/hierarchy_top.json (hunk 2 above)
Comment (shifucun) [reply static/data/hierarchy_top.json]: Done

**Review by beets — APPROVED** [review]

**Pull request merged**